scw/custom_types.go

```diff
@@ -84,24 +88,28 @@ func NewMoneyFromFloat(value float64, currency string) *Money {
 
 // String returns the string representation of Money.
 func (m *Money) String() string {
-	defaultCurrencySign := "€"
 	currencySignsByCodes := map[string]string{
-		"EUR": defaultCurrencySign,
+		"EUR": "€",
 		"USD": "$",
 	}
 
 	currencySign, currencySignFound := currencySignsByCodes[m.CurrencyCode]
 	if !currencySignFound {
-		currencySign = defaultCurrencySign
+		logger.Debugf("%s currency code is not supported", m.CurrencyCode)
+		currencySign = m.CurrencyCode
 	}
 
-	str := fmt.Sprintf("%s %d", currencySign, m.Units)
-
+	value := fmt.Sprint(m.Units)
```

Comment:
You can entirely rely on float default formatting for this: https://play.golang.org/p/6Xa73sKvE4e